airgun/entities/discoveryrule.py

```diff
@@ -0,0 +1,102 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.discoveryrule import (
+    DiscoveryRuleCreateView,
+    DiscoveryRuleEditView,
+    DiscoveryRulesView,
+)
+from airgun.views.host import HostsView
+
+
+class DiscoveryRuleEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        view.submit.click()
```

Comment:
Don't forget to assert notifications and dismiss them for all actions which trigger flash notifications